Resources/views/GraphiQL/index.html.twig

```diff
@@ -46,7 +46,7 @@
         var xdebugToken = res.headers.get('X-Debug-Token')
         if (typeof Sfjs !== "undefined" && xdebugToken) {
           var toolbarElement = document.querySelector('.sf-toolbar')
-          var debugUrlPattern = "{{ url('_wdt', {'token': '__TOKEN__'}) }}"
+          var debugUrlPattern = "{{ app.debug ? url('_wdt', {'token': '__TOKEN__'}) : '/_wdt/__TOKEN__' }}"
```

Comment:
Yeah we can wrap the `if` (javascript) in a `if app.debug` (twig) :) 

The reason why I didn't do it was because `app.debug` doesn't necessarily mean that there is no web debug toolbar. I was trying to find a good way to test for that but didn't see a good way. Best would be IMO to somehow test for that. Do you have an idea?